articles/article-28-machine-language-vs-programming-language.md

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN TITLE -->
+# Machine Language vs. Programming Language
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+A programming language is a language that programmers use to develop applications. It is also called high-level language as they are more human readable.  Examples of such programming language include -but not limited to- C++, Java, and Fortran.
```

Comment:
The way it is currently written, the text says that every programming language is a high-level language. 
We might need to add a sentence here that says that we distinguish between high-level and low-level languages.

Reply:
Language suggestion: Instead of -but not limited to- one could just write "include for example". IMO this makes it more clear that there are also others